CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v0.13.0 - 2023-11-13
 
+### enhancement
+- Create E2E resources Helm chart by @juanjjaramillo in [#377](https://github.com/newrelic/newrelic-infra-operator/pull/377)
```

Comment:
Can you move this to be under the ## Unreleased heading please?

Reply:
Nice catch! Done